resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+################
+# Usage: controller.sh <os>
+# The controller script does all of the jar compilation
+# and setup needed to build packages for <os>
+# The os definition is fairly flexible using string matching
+# and currently supports el6, el7, sles11, sles12, debian jessie,
+# debian stretch, ubuntu trusty, ubuntu xenial, and ubuntu yakkety
+################
+build_os=$1
+build_ver=$2
+
+BASEPATH='pkg_artifacts'
+mkdir $BASEPATH
+if [ ! -d "$BASEPATH/base" ]; then
+        DESTDIR="$BASEPATH/base" bash install.sh install_redhat
```

Comment:
The only difference between install_deb and install_redhat is if there are docs (whether or not to be in a versioned directory), so if there aren't any docs installed with the package there's no need to run it again